services/api/mw2pdf/utils/validation.js

```diff
@@ -0,0 +1,45 @@
+import Joi from 'joi'
+
+function isObject(value) {
+  return (typeof value === 'object' && value !== null)
+}
+
+const validJsonStringSchema = Joi.string()
```

Comment:
Just a note that I think the most "clean" way to do this would be to actually have schemas in their own directory, with each validation schema defined in its own file.

So we could do things like...

```
import {
  validJsonStringSchema,
  passthroughParameterSchema,
} from './schemas'
```

The only reason I'm not doing that right now is that I would like to make that improvement after we move to TypeScript.  But @hawkinsw if you think I should just make it happen I'd be fine to take that step as part of this PR.